integration-tests/tests/create-application-from-public-git-source.spec.ts

```diff
@@ -41,12 +44,9 @@ describe('Create Component from Public Git Source', { tags: ['@PR-check', '@publ
       Common.waitForLoad();
```

Comment:
IDK, is this wait needed there? We are still on the same page.

Reply:
removed